yb-voyager/cmd/common.go

```diff
@@ -802,7 +809,7 @@ func renameTableIfRequired(table string) (string, bool) {
 	fromTable := tableName.Qualified.Unquoted
 
 	if renameTablesMap[fromTable] != "" {
-		table := sqlname.NewSourceNameFromQualifiedName(renameTablesMap[fromTable])
+		table := sqlname.NewSourceNameFromQualifiedName(getQuotedFromUnquoted(renameTablesMap[fromTable]))
```

Comment:
Yes, but I had to use sqlname.SourceName at other places for leaf partitions to avoid manual NameTuple creation so thought to use sqlname.SourceName in these few places to avoid extra lookup which is not required because we anyway lookup when we read dataFileDescriptor.